frontend/src/metabase/query_builder/selectors.js

```diff
@@ -13,7 +13,10 @@ import { parseTimestamp } from "metabase/lib/time";
 import { getSortedTimelines } from "metabase/lib/timelines";
 import { isNotNull } from "metabase/lib/types";
 import { getAlerts } from "metabase/notifications/redux/selectors";
-import { getEmbedOptions, getIsEmbedded } from "metabase/selectors/embed";
+import {
```

Comment:
there was a conflict here because the line above isn't the same. We got `import { isNotNull } from "metabase/lib/types";` above this line in the original PR.